CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ if(LUAU_STATIC_CRT)
     set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
 endif()
 
-project(Luau LANGUAGES CXX C)
+project(Luau LANGUAGES CXX C VERSION 0.607)
```

Comment:
I don't think we can include a version.
Even with automation (worse with manual update) it will require a commit to this CMake file basically each week.